com.unity.render-pipelines.universal/Editor/ShaderPreprocessor.cs

```diff
@@ -237,7 +233,7 @@ public void OnProcessShader(Shader shader, ShaderSnippetData snippetData, IList<
 
 
             int prevVariantCount = compilerDataList.Count;
-           
+            Debug.Log(ShaderBuildPreprocessor.supportedFeatures);
```

Comment:
Is this Debug supposed to still be here?

Reply:
Yes I saw it just when I pushed the commit. :)